### PR TITLE
Feature/inform user about negative ages

### PR DESCRIPTION
**Pull request by aj-thl**
Added notification about negative values in age. Also, negative values in age are now transformed to NA.

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -26,6 +26,7 @@
 * Improved wording of objects in signals and reports tab
 * Made table appearance for interactive and static tables consistent using new dependency `flextable`
 * Recodes NAs in signal tables to 'unknown' for all stratifications but not for the unstratified level
+* Added notification about negative values in age.
```
Comment (tinneuro) [inline NEWS.md:29]: same here as for the other PR. Can you move the added change to the new section of NEWS?

File: R/mod_tabpanel_data.R [hunk 2]
```diff
@@ -62,8 +62,8 @@ mod_tabpanel_data_ui <- function(id) {
           htmlOutput(ns("unused_vars"))
         ),
         hr(),
-        h4("Cases which have missing data"),
-        span(paste0("In this section you receive feedback about cases for which data required for the computations is missing and will thus be removed for all analysis. The following variables are checked for missingness: ",check_for_missing_values(), ". If any of these variables have a missing value the case is removed for all analysis.")),
+        h4("Data Quality"),
```
Comment (tinneuro) [inline R/mod_tabpanel_data.R:65]: I like that change thanks!

**Comment by tinneuro** [comment]
I am not done yet with the full review. Will do the rest tomorrow morning

File: NEWS.md (hunk 1 above)
Comment (aj-thl) [reply NEWS.md:29]: Of course, I will do this.

File: R/mod_tabpanel_data.R [hunk 3]
```diff
@@ -62,8 +62,8 @@ mod_tabpanel_data_ui <- function(id) {
...
+        span(paste0("In this section you receive feedback about data quality and cases for which data required for the computations is missing and will thus be removed for all analysis. The following variables are checked for missingness: ",check_for_missing_values(), ". If any of these variables have a missing value the case is removed for all analysis.")),
```
Comment (tinneuro) [inline R/mod_tabpanel_data.R:66]: Maybe we can change the text to the following which is a bit easier to read:
"In this section you receive feedback about implausible or inconsistent values and missing data. The following variables are checked for missingness: ",check_for_missing_values(), ". The following variables are checked for inconsistent/implausible values: age.

File: R/mod_tabpanel_data.R [hunk 4]
```diff
@@ -72,6 +72,7 @@ mod_tabpanel_data_ui <- function(id) {
           shiny::uiOutput(ns("missing_vals"))
         ),
         hr(),
+        shiny::uiOutput(ns("negative_ages")),
```
Comment (tinneuro) [inline R/mod_tabpanel_data.R:75]: I like that in your message it is directly included what will happen with the negative ages. Maybe can you just adapt this for the case_id message as well that there it is written that these Case IDs are excluded from analysis. Then we can remove it in line 68/69

**Comment by tinneuro** [comment]
I have only minor comments. :)

File: R/mod_tabpanel_data.R (hunk 3 above)
Comment (aj-thl) [reply R/mod_tabpanel_data.R:66]: I agree, what you are suggesting is better!

File: R/mod_tabpanel_data.R (hunk 4 above)
Comment (aj-thl) [reply R/mod_tabpanel_data.R:75]: Sure, I'll see what I can do about it.

Comment (tinneuro) [reply R/mod_tabpanel_data.R:75]: If not no worries. Just a minor thing

Comment (aj-thl) [reply R/mod_tabpanel_data.R:70]: Ok I just adjusted the code as you requested and pushed the changes. Please tell what you think of it :)

Comment (tinneuro) [reply R/mod_tabpanel_data.R:70]: Thanks, it is good. Have a nice and well deserved holiday!

**Review by tinneuro — APPROVED** [review]

File: R/mod_tabpanel_data.R (hunk 4 above)
Comment (aj-thl) [reply R/mod_tabpanel_data.R:70]: Thank you! :)

**Pull request merged**